src/main/java/duke/program/Launcher.java

```diff
@@ -0,0 +1,12 @@
+package duke.program;
+
+import javafx.application.Application;
+
+/**
+ * A launcher class to workaround classpath issues.
+ */
+public class Launcher {
+    public static void main(String[] args) {
+        Application.launch(Main.class, args);
+    }
+}
```

Comment:
No newline at the end of file. Small error, but integrating some automated checkstyle might alleviate some styling issues.